### ISSUE TITLE
release a new version

### ISSUE THREAD

**Issue by olivierchatry** [lazybensch/ember-cli-copyable#24]
Would be cool to have a new version with all the fixes :).

**Comment by milindalvares** [comment]
````
DEPRECATION: Addon files were detected in `/Users/milindalvares/dev/invoicesapp-frontend-ember/node_modules/ember-cli-copyable/addon`, but no JavaScript preprocessors were found for `ember-cli-copyable`. Please make sure to add a preprocessor (most likely `ember-cli-babel`) to in `dependencies` (NOT `devDependencies`) in `ember-cli-copyable`'s `package.json`.

````

Errors from e-cli-2.12.0

**Comment by olivierchatry** [comment]
Well it works for me with 2.12. ( and ember-cli-babel is in the dependency of this addon ). Maybe you should try to remove your node_modules folder and `npm install` again ?

**Comment by milindalvares** [comment]
Sure you aren't getting a deprecation warning when you start the Ember server?

**Comment by basz** [comment]
seeing the same with 2.12.1

**Comment by olivierchatry** [comment]
Yep, I had, and I also did not saw you pushed a PR, so I did one. I guess the only difference is I removed the dependency from devDependencies. Ho well.

**Comment by gogromat** [comment]
same on 2.12.1

**Comment by haldiggs** [comment]
this fixes it... I cannot do a pull from her though... please add

  "dependencies": {
  "ember-cli-babel": "^4.0.0"
  },
  "devDependencies": {
    "broccoli-asset-rev": "^2.0.0",
    "ember-cli": "0.2.0",
    "ember-cli-app-version": "0.3.2",
    "ember-cli-content-security-policy": "0.3.0",
    "ember-cli-dependency-checker": "0.0.8",
    "ember-cli-htmlbars": "0.7.4",
    "ember-cli-ic-ajax": "0.1.1",
    "ember-cli-inject-live-reload": "^1.3.0",
    "ember-cli-qunit": "0.3.9",
    "ember-cli-uglify": "1.0.1",
    "ember-data": "1.0.0-beta.15",
    "ember-export-application-global": "^1.0.2"
  },

**Comment by wuarmin** [comment]
I have the same issue with Ember 2.12.1. It would be nice to get rid of it :+1: 
Thanks!

**Comment by Falibur** [comment]
Would love to see a new version with the fixes!

**Comment by ghost** [comment]
see #28 for the discussion.
